packages/gamemode/src/playerActions/quickChat.ts

```diff
@@ -0,0 +1,34 @@
+import { takeLatest, put } from "@redux-saga/core/effects";
+import { createAction } from "@reduxjs/toolkit";
+import { getDependency } from "@shoki/engine/lib";
```

Comment:
This import should just be `@shoki/engine` - basically avoid `/lib` in any imports as it's probably not correct (and can break at runtime... but only sometimes. i dont really understand when vs when not)